examples/main.tf

```diff
@@ -275,7 +275,7 @@ resource "instaclustr_cluster" "example-opensearch" {
 
   bundle {
     bundle = "OPENSEARCH"
-    version = "1.3.4"
+    version = "1.3.5"
     options = {
       dedicated_master_nodes = true,
       master_node_size = "SRH-DEV-t4g.small-5",
```

Comment:
can you also test and fix this example as it's incorrect, i.e. dedicated master node size is wrong

Reply:
Thanks Long, updated

Reply:
all good
you can have this high ticket resolved with this fix
https://instaclustr.atlassian.net/browse/INS-20681